.github/workflows/release-upload.yml

```diff
@@ -0,0 +1,83 @@
+name: Build and upload release assets
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  release_assets:
+    name: Release assets
+    runs-on: ${{ matrix.config.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+        # Linux
+        - {os: ubuntu-latest, toolchain: stable, target: x86_64-unknown-linux-gnu, cross: false}
+        - {os: ubuntu-latest, toolchain: stable, target: aarch64-unknown-linux-gnu, cross: true}
+        - {os: ubuntu-latest, toolchain: stable, target: arm-unknown-linux-gnueabi, cross: true}
+        - {os: ubuntu-latest, toolchain: stable, target: armv7-unknown-linux-gnueabihf, cross: true}
+        # macOS
+        - {os: macos-latest, toolchain: stable, target: x86_64-apple-darwin, cross: false}
+        # Windows
+        - {os: windows-latest, toolchain: stable-x86_64-pc-windows-msvc, target: x86_64-pc-windows-msvc, cross: false}
+
+    steps:
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.config.toolchain }}
+          target: ${{ matrix.config.target }}
+          override: true
+          default: true
+
+      - name: Checkout code
+        uses: actions/checkout@v1
+        with:
+          submodules: true
+
+      - name: Get version tag
+        if: matrix.config.os != 'windows-latest'
```

Comment:
Get version will fail for windows. I think is safe to release the if

Reply:
I was not going to use it for Windows, because the binary name should've been just `./jorup-init.exe`.

Reply:
removed this line anyway, doesn't make much sense